src/Parts/ApplicationCommandOptionStructure.php

```diff
@@ -22,15 +22,15 @@ class ApplicationCommandOptionStructure
     public ?array $description_localizations;
     public ?bool $required;
     /**
-     * @var \Ragnarok\Fenrir\Parts\ApplicationCommandOptionChoice[]
+     * @var ApplicationCommandOptionChoice[]
      */
     public ?array $choices;
     /**
-     * @var \Ragnarok\Fenrir\Parts\ApplicationCommandOptionStructure[]
+     * @var ApplicationCommandOptionStructure[]
      */
     public ?array $options;
     /**
-     * @var \Ragnarok\Fenrir\Enums\Parts\ChannelTypes[]
+     * @var ChannelTypes[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)